pom.xml

```
@@ -23,12 +23,25 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-data-jpa</artifactId>
 		</dependency>
-		
+		<dependency>
+       <groupId>com.alibaba</groupId>
```

```suggestion
       <groupId>com.alibaba</groupId>
```
```suggestion
       <groupId>com.alibaba</groupId>
```
```suggestion
       <groupId>com.alibaba</groupId>
```
```suggestion
       <groupId>com.alibaba</groupId>

```suggestion
       <groupId>com.alibaba</groupId>
```